etcdserver/server.go

```diff
@@ -602,7 +602,41 @@ func (s *EtcdServer) LeaderStats() []byte {
 
 func (s *EtcdServer) StoreStats() []byte { return s.store.JsonStats() }
 
+func isReadyToAddNewMember(members map[types.ID]*Member) bool {
+	nmembers := 1
+	nstarted := 0
+
+	for _, member := range members {
+		if len(member.Name) != 0 {
```

Comment:
One more tip: i would like to make this a member function because it is implicit that `len(member.Name) != 0` -> unstarted member`